tools/text_processing/text_processing/tool_dependencies.xml

```diff
@@ -1,34 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
I have not planned to upload this one to the MTS until we have discussed the "recommended" system at GCC.
I hope this is ok. I could leave the dependencies in, but the version do not match anymore and it would mean new tool_dependencies.